package.json

```diff
@@ -20,14 +19,8 @@
     "url": "https://github.com/hareko/js-merge-xml/issues"
   },
   "devDependencies": {
-    "chai": "3.0.x",
```

Comment:
There is some mess with the github and my local package test versions. I'll send by email -
please check before including again.

Reply:
Simply add these 4 modules back as devDependencies + the files you removed, and all should be good.

(Just `rm -R node_modules` (and perhaps rm package-lock.json) and re-run `npm install` to clear up any package mess

Reply:
What 4 modules to add back? This is in your package.json:
"devDependencies": {
    "chai": "3.0.x",
    "karma": "1.3.x",
    "karma-chai": "0.1.x",
    "karma-chrome-launcher": "2.0.x",
    "karma-firefox-launcher": "1.0.x",
    "karma-mocha": "1.2.x",
    "karma-opera-launcher": "1.0.x",
    "karma-safari-launcher": "1.0.x",
    "mocha": "3.1.x"
}

Reply:
Oh sorry, I was looking at the diff and saw only 4. All of the ones you listed above should go back (and since they are `devDependencies`, don't worry about how full node_modules looks). It's only `dependencies` that really matter.